README.md

```diff
@@ -2,39 +2,49 @@
  
 # Table of Contents
 
-* [Allgemeines](#allgemeines)
-   * [Schaufensterkonditionen](#schaufensterkonditionen)
-* [Beispiele](#beispiele)
-   * [Query topSchaufensterkondition](#query-topschaufensterkondition)
-      * [POST Request](#post-request)
-      * [POST Response](#post-response)
-   * [Query schaufensterkonditionen](#query-schaufensterkonditionen)
-      * [POST Request](#post-request-1)
-      * [POST Response](#post-response-1)
-* [Request](#request)
-   * [Authentifizierung](#authentifizierung)
-   * [Nachverfolgbarkeit von Requests](#nachverfolgbarkeit-von-requests)
-   * [Format](#format)
-   * [Request Parameter](#request-parameter)
-      * [Query topSchaufensterkondition](#query-topschaufensterkondition-1)
-      * [Query schaufensterkonditionen](#query-schaufensterkonditionen-1)
-      * [Partner-ID](#partner-id)
-      * [Datenkontext](#datenkontext)
-      * [Finanzierungszweck](#finanzierungszweck)
-   * [Anfragbare Felder](#anfragbare-felder)
-      * [Schaufensterkondition](#schaufensterkondition)
-         * [Gesamtkonditionen](#gesamtkonditionen)
-            * [Konditionsspanne](#konditionsspanne)
-              * [Konditionsgrenze](#konditionsgrenze)
-         * [Ratenkredit](#ratenkredit)
-            * [Produktanbieter](#produktanbieter)
-            * [Anschrift](#anschrift)
-            * [Logo](#logo)
-* [Fehlercodes](#fehlercodes)
-   * [HTTP-Status Errors](#http-status-errors)
-   * [Weitere Fehler](#weitere-fehler)
-* [Tools](#tools)
-* [Nutzungsbedingungen](#nutzungsbedingungen)
+- [Allgemeines](#allgemeines)
+  * [Authentifizierung](#authentifizierung)
+  * [Nachverfolgbarkeit von Requests](#nachverfolgbarkeit-von-requests)
+  * [Request](#request)
+  * [Fehlercodes](#fehlercodes)
+    + [HTTP-Status Errors](#http-status-errors)
+    + [Weitere Fehler](#weitere-fehler)
+  * [Tools](#tools)
+- [Schaufensterkonditionen](#schaufensterkonditionen)
+  * [Query Top-Schaufensterkondition](#query-top-schaufensterkondition)
+    + [Request](#request-1)
+    + [Response](#response)
+    + [Beispiel](#beispiel)
+      - [POST Request](#post-request)
+      - [POST Response](#post-response)
+  * [Query Schaufensterkonditionen](#query-schaufensterkonditionen)
+    + [Request](#request-2)
+    + [Response](#response-1)
+    + [Beispiel](#beispiel-1)
+      - [POST Request](#post-request-1)
+      - [POST Response](#post-response-1)
+- [Angebote](#angebote)
+  * [Query Angebote ermitteln](#query-angebote-ermitteln)
```

Comment:
Hinsichtlich Symmetrie zu den Schaufensterkonditionen: das `ermitteln` kann weg? "Query" ist ja als Begriff schon "Abfragen/Ermitteln".

Reply:
nehme ich erstmal raus war aber schon in die Richtung, dass wir auch noch was Richtung "annehmen" zur Verfügung stellen müssen.

Reply:
Ok, dann wird's eh wieder spezifisch.